pkg/modules/libreoffice/routes.go

```diff
@@ -32,6 +32,7 @@ func convertRoute(libreOffice libreofficeapi.Uno, engine gotenberg.PdfEngine) ap
 				singlePageSheets         bool
 				losslessImageCompression bool
 				reduceImageResolution    bool
+				exportNotesInMargin      bool
```

Comment:
Could you also move this before `LosslessImageCompression`?

Reply:
Done